js/sound.js

```diff
@@ -198,3 +198,13 @@ function playAudio(audio, licenseNum) {
     audioPlay.play();
     $("#tileLicense").html(creativeCommonsLicense[licenseNum - 1]);
 }
+
+function playPauseFloat() {
+    if(backgroundPlay.paused === true) {
+        startBackgroundAudio();
```

Comment:
What if someone calls "startBackgroundAudio()" Would your button change its picture? If you like some software architecture input, please see this as way to notify others about the change: https://en.wikipedia.org/wiki/Observer_pattern

Reply:
Ohh. So, if someone calls it by clicking the buttons on the sidebar, it doesn't change the picture. On it!

Reply:
Done!